docs/meshstack.openshift.index.md

```diff
@@ -218,8 +218,65 @@ roleRef:
   name: meshfed-service
 ```
 
+### Custom meshProject Roles
+
+If you want to use custom roles to be mapped to your meshProject roles (and not just the pre-defined `admin`, `edit` and `view` roles) you need to make sure to also list these roles in the clusterrole binding section for the meshfed-service principle. It is not allowed for the service-principle to bind roles granting more rights then itself has, so the right to bind these roles must be explicitly given.
+
+For example if you plan to use a role named `my-custom-role` please change the relevant section in the above document to:
+
+```yml
+- apiGroups:
+  - ""
+  - rbac.authorization.k8s.io
+  - authorization.openshift.io
+  resources:
+  - clusterroles
+  verbs:
+  - bind
+  resourceNames:
+  - admin
+  - edit
+  - view
+  - my-custom-role
+```
+
 #### Metering
 
+Next, retrieve the access token for the service account:
```

Comment:
f: Everything from here on must be deleted. It is currently duplicated. LZ is now defined in separate section and the way how to access the service token has changed a bit. This was just recently merged, so something went wrong during merge or so